docs/pages/api-reference/operators.md

```diff
@@ -101,6 +101,19 @@ The `aggregate` operator has the following parameters:
 
 <pre snippet="api-reference/operators_ref#aggregate"></pre>
 
+### Groupby/ First
```

Comment:
Please add a code snippet example?

Reply:
Missed that. Will add something

Reply:
Added an example. I am not able to visualize the docs on my local environment. If your setup is working, can you preview and make sure it looks good?